lib/jellyfish_web/api_spec/component/file.ex

```diff
@@ -0,0 +1,52 @@
+defmodule JellyfishWeb.ApiSpec.Component.File do
+  @moduledoc false
+
+  require OpenApiSpex
+  alias OpenApiSpex.Schema
+
+  defmodule Properties do
+    @moduledoc false
+
+    require OpenApiSpex
+
+    OpenApiSpex.schema(%{
+      title: "ComponentPropertiesFile",
+      description: "Properties specific to the File component",
+      type: :object,
+      properties: %{}
+    })
+  end
+
+  defmodule Options do
+    @moduledoc false
+
+    require OpenApiSpex
+    alias OpenApiSpex.Schema
+
+    OpenApiSpex.schema(%{
+      title: "ComponentOptionsFile",
+      description: "Options specific to the File component",
+      type: :object,
+      properties: %{
+        filePath: %Schema{
+          type: :string,
+          description: "Path to track file. Must be either OPUS or h264",
```

Comment:
Correct me if I'm wrong, but I think this should be specified

```suggestion
          description: "Path to track file. Must be either OPUS encapsulated in OGG or raw h264",
```